<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 32, 28);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="question">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>771</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt &quot;BoyzRGross&quot;;
color: rgb(0, 255, 53);</string>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="answer1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(93, 255, 65);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-radius: 15%;
border: 4px solid rgb(93, 255, 65);</string>
    </property>
    <property name="text">
     <string>Answer 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(93, 255, 65);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-radius: 15%;
border: 4px solid rgb(93, 255, 65);</string>
    </property>
    <property name="text">
     <string>Answer 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(93, 255, 65);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-radius: 15%;
border: 4px solid rgb(93, 255, 65);</string>
    </property>
    <property name="text">
     <string>Answer 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(93, 255, 65);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-radius: 15%;
border: 4px solid rgb(93, 255, 65);</string>
    </property>
    <property name="text">
     <string>Answer 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
